### PR TITLE
Feature/jac fetcher migration

### PR DESCRIPTION
**Pull request by napalmpapalam**

### CODE REVIEW

**Review by lukachi — APPROVED** [review]

File: packages/fetcher/README.md [hunk 1]
```diff
@@ -33,33 +33,34 @@ In HTML:
 ```
```
Comment (ArtSuslov) [inline packages/fetcher/README.md:33]: fix formatting

File: packages/jac/src/errors/server-errors.ts [hunk 2]
```diff
@@ -7,127 +7,130 @@ import {
...
+    const detail = unwrappedError?.detail ?? ''
+    const title = unwrappedError?.title ?? ''
+
+    this.message = `${title}: ${detail}`
+
+    this.#originalError = originalError
+    this.#meta = unwrappedError?.meta ?? {}
+
+    if (errors.length > 1) {
+      this.message = 'Request contains some errors. Check "nestedErrors"'
```
Comment (ArtSuslov) [inline packages/jac/src/errors/server-errors.ts]: some -> multiple

File: packages/jac/src/json-api.ts [hunk 3]
```diff
@@ -150,15 +175,15 @@ export class JsonApiClient {
    * Parses the response in JsonApi format.
    */
   post<T, U = JsonApiDefaultMeta>(
-    endpoint: string,
-    data: unknown,
-    isNeedRaw?: boolean,
+    endpoint: Endpoint,
+    body?: JsonApiClientRequestBody,
```
Comment (ArtSuslov) [inline packages/jac/src/json-api.ts:179]: is it allowed to send `{data: undefined}`?

**Review by ArtSuslov — CHANGES_REQUESTED** [review]

File: packages/jac/src/json-api.ts (hunk 3 above)
Comment (napalmpapalam) [reply packages/jac/src/json-api.ts:179]: yes, u can have a post endpoint which doesn't require body

File: packages/jac/src/errors/server-errors.ts (hunk 2 above)
Comment (napalmpapalam) [reply packages/jac/src/errors/server-errors.ts]: fixed

File: packages/fetcher/README.md (hunk 1 above)
Comment (napalmpapalam) [reply packages/fetcher/README.md:33]: fixed

**Review by ArtSuslov — APPROVED** [review]

**Pull request merged**